<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1449</width>
    <height>1359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QGroupBox" name="gB_tof_options">
     <property name="title">
      <string>Tof Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QPushButton" name="pB_get_wavelength">
        <property name="text">
         <string>get from data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="SB_epp_channel">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="l_wavelength">
        <property name="text">
         <string>Wavelength</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_epp_corr">
        <property name="text">
         <string>Elastic peak channel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="dSB_wavelength">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>4.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="cB_delete_raw">
        <property name="text">
         <string>delete raw workspaces</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="gB_orientation">
     <property name="title">
      <string>Orientation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="l_hkl1">
        <property name="toolTip">
         <string>First reciprocal lattice vector in the scattering plane</string>
        </property>
        <property name="text">
         <string>hkl1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lE_u">
        <property name="text">
         <string>1,0,0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lE_v">
        <property name="text">
         <string>0,0,1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_hkl2">
        <property name="toolTip">
         <string>Second reciprocal lattice vector in the scattering plane, should be perpendicular hkl1</string>
        </property>
        <property name="text">
         <string>hkl2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_omega_offset">
        <property name="text">
         <string>omega offset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dSB_omega_offset">
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="gB_unit_cell">
     <property name="title">
      <string>Unit cell</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="7" column="0">
       <widget class="QLabel" name="l_alpha">
        <property name="text">
         <string>alpha</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="dSB_beta">
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="l_gamma">
        <property name="text">
         <string>gamma</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_b">
        <property name="text">
         <string>b</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="dSB_gamma">
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="dSB_alpha">
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="dSB_c">
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dSBl_b">
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="l_beta">
        <property name="text">
         <string>beta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_a">
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="l_c">
        <property name="text">
         <string>c</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="dSB_a">
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="5">
    <widget class="QGroupBox" name="gB_corrections">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Corrections</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="12" column="0" colspan="3">
       <widget class="QCheckBox" name="cB_correct_elastic_peak_position">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Correct elastic peak position to 0 meV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="4">
       <widget class="QCheckBox" name="cB_substract_background">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Substract Bacground from sample</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="dSB_background_factor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.800000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="4">
       <widget class="QCheckBox" name="cB_substract_background_from_vanadium">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Subtract instrument background from vanadium</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QCheckBox" name="cB_det_efficency">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Detector efficiency correction</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="dSB_vanadium_temperature">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>290.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="l_background_factor">
        <property name="text">
         <string>     Factor</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="l_vanadium_temperature">
        <property name="text">
         <string>     Vanadium temperature</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="4">
       <widget class="QCheckBox" name="cB_mask_bad_detectors">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Mask bad detectors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gB_normalisation">
     <property name="title">
      <string>Normalisation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="cB_monitor_normalization">
        <property name="text">
         <string>Normalize to Monitor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="gB_binning">
     <property name="title">
      <string>Binning</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="dSB_hmin">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="l_hmax">
        <property name="text">
         <string>h max</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="l_hmin">
        <property name="text">
         <string>h min</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="dSB_hmax">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_kmin">
        <property name="text">
         <string>k min</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="dSB_lmin">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="dSB_dEmin">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="value">
         <double>-5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="l_dEmin">
        <property name="text">
         <string>dE min</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_lmin">
        <property name="text">
         <string>l min</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dSB_kmin">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="l_hbins">
        <property name="text">
         <string>h bins</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="dSB_kmax">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="l_kmax">
        <property name="text">
         <string>k max</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="l_lmax">
        <property name="text">
         <string>l min</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="l_dEmax">
        <property name="text">
         <string>dE max</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="dSB_lmax">
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="dSB_dEmax">
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLabel" name="l_lbins">
        <property name="text">
         <string>l bins</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="l_kbins">
        <property name="text">
         <string>k bins</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="l_dE_bins">
        <property name="text">
         <string>dE bins</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="sB_hbins">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QSpinBox" name="sB_kbins">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QSpinBox" name="sB_lbins">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QSpinBox" name="sB_dEbins">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="5">
       <widget class="QPushButton" name="pB_estimate">
        <property name="text">
         <string>suggest from data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
